Source/WebCore/rendering/RenderWidget.cpp

```diff
@@ -228,8 +228,10 @@ void RenderWidget::styleDidChange(StyleDifference diff, const RenderStyle* oldSt
         else
             m_widget->show();
 
-        if (CheckedPtr cache = document().existingAXObjectCache())
+        if (CheckedPtr cache = document().existingAXObjectCache()) {
             cache->onWidgetVisibilityChanged(*this);
+            cache->childrenChanged(this);
```

Comment:
Does this method (`RenderWidget::styleDidChange`) when any style on the iframe changes, even ones that don't affect the accessibility tree at all (which would be most of them)? If so, that would mean we're going to issuing potentially a lot more children changed events, most unnecessary.

Reply:
This was only needed for the inert attribute, because we weren't handling that in `RenderElement::styleWillChange`. I'm updating `RenderElement::styleWillChange` now to include a check for `inert` changing, like we do for visibility, so we can remove childrenChanged here and only call it once there (and only for those two attributes changing.